### PR TITLE
Fix Issue Expose size of MetricBuffer

### PR DESCRIPTION
**Pull request by Yogendra0Sharma**
Fix Issue #213

### CODE REVIEW

File: telemetry_core/src/main/java/com/newrelic/telemetry/metrics/MetricBuffer.java [hunk 1]
```diff
@@ -47,6 +47,14 @@ public void addMetric(Metric metric) {
     metrics.add(metric);
   }
 
+  /**
+   * Get the size of the {@link Metric Metrics} buffer.
+   *
+   * @return Size of the {@link Metric Metrics} buffer.
+   */
+  public int size() {
```
Comment (jkwatson) [inline telemetry_core/src/main/java/com/newrelic/telemetry/metrics/MetricBuffer.java:55]: Can you add a unit test to verify this works? Thanks!

Comment (Yogendra0Sharma) [reply telemetry_core/src/main/java/com/newrelic/telemetry/metrics/MetricBuffer.java:55]: @jkwatson Where to add unit testing.  the repo contains only an integration test, do I have to add a new test into the integration_test folder suite ?

Comment (jkwatson) [reply telemetry_core/src/main/java/com/newrelic/telemetry/metrics/MetricBuffer.java:55]: The repo contains many many tests in the src/test directory. Please add a test case to the `MetricBufferTest` for this change.

File: telemetry_core/src/test/java/com/newrelic/telemetry/metrics/MetricBufferTest.java [hunk 2]
```diff
@@ -38,6 +38,19 @@ void testMetrics() {
     assertEquals(expectedGauge, metricBuffer.getMetrics().poll());
   }
 
+  @Test
+  @DisplayName("Get size of Metrics")
+  void testMetricsSize() {
+    Attributes expectedCommonAttributes =
+        new Attributes().put("key1", "value1").put("key2", true).put("key3", 123.456);
+
+    MetricBuffer metricBuffer = new MetricBuffer(expectedCommonAttributes);
```
Comment (jkwatson) [inline telemetry_core/src/test/java/com/newrelic/telemetry/metrics/MetricBufferTest.java:47]: can you add an assertion that the size is `0` before you add one? Then, this looks great. Thanks!

**Comment by breedx-nr** [comment]
Just to mention, I also filed #216 as the Event equivalent for this issue.

**Pull request merged**